lib/filters/global/actions.js

```diff
@@ -17,7 +17,12 @@ module.exports = {
                     body.format = 'json';
                     return restbase[req.method](req)
                     .then(function(res) {
-                        if (res.status !== 200) {
+                        if (
+                            res.status !== 200
```

Comment:
This would be simplified if we raised exceptions / rejected the promise like in preq.